decentraland/unity-renderer
# Spike: test world with base main camera

## What does this PR change?

Fix #3583
Changes main camera from overlay to base

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=spike/base-main-camera
2. Check around the world and verify everything is correctly displayed
3. Focus more on areas with transparency, scenes with complex UI and such
4. Verify the skybox is always correctly rendered at the back of everything

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
